src/greeter/GreeterApp.h

```diff
@@ -38,7 +38,7 @@ namespace SDDM {
     class KeyboardModel;
 
 
-    class GreeterApp : public QGuiApplication
+    class GreeterApp : public QApplication
```

Comment:
I see why you're doing this, but I'd really like not to. 

QApplication is huge compared to QGuiApplication, it'll add half a second to bootup time easily.
and because you don't have a QtPlatformTheme loaded, it'll look hideous too.

Don't you have another patch that exposes messages from the sysadmin into the UI? Can't we share that?

We'll need a solution to stop the greeter from closing, lightDM used to solve that by not logging in when authentication was complete but instead waiting for a signal from the greeter.